src/test/java/com/training/base/BaseTest.java

```diff
@@ -28,7 +25,7 @@ public void setUp() {
         options.addArguments("--incognito");
         options.addArguments("start-maximized");
         Configuration.browserCapabilities.setCapability(ChromeOptions.CAPABILITY, options);
-        map = (Map) new GetJsonValue().getValue();
```

Comment:
add this removed line to @beforeAll block in this class, or else it won't work for existing tests